terraso_backend/apps/auth/services.py

```diff
@@ -108,3 +112,89 @@ def _get_base_payload(self, user):
             "jti": uuid4().hex,
             "email": user.email,
         }
+
+
+class PlausibleService:
+    """Service for making API calls to plausible service.
+
+    See documentation at https://plausible.io/docs/events-api .
+    """
+
+    PLAUSIBLE_URL = "https://plausible.io/api/event"
```

Comment:
This is the only instance of the API, and there's no way that we could run a local instance so the value isn't going to change. However it could make sense to keep these kinds of values outside of the code in general, it is a bit neater. I think I will move the value to the settings, unless you have some objections.

Reply:
Plausible is open source and can be self-hosted. https://github.com/plausible/analytics